python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -777,14 +776,515 @@ class ReferenceObjectLoader(ReferenceObjectLoaderBase):
         Logger object used to write out messages. If `None` a default
         logger will be used.
     """
+    ConfigClass = LoadReferenceObjectsConfig
+
     def __init__(self, dataIds, refCats, log=None, config=None, **kwargs):
```

Comment:
Can we get rid of the `**kwargs`, since they're not used?

Reply:
I'm worried that could break some existing code (if somebody was erroneously passing a `butler=` for example).  The problem was that usage was incorrect but never warned.  So what I'm going to do is check if `**kwargs` is non-empty and throw up a deprecation warning.